### DISCUSSION TITLE
Export Javascript

### DISCUSSION THREAD

**Discussion by Lahirutech** [GrapesJS/grapesjs#3519] [Q&A]
Can someone help me to find a way how to export js code related to a component using the export plugin/or any other way? Like onclick methods related to a particular component.

or At least a place to look in the documentation.

**Comment by artf** [comment]
You can do this at the moment:
```js
const component = editor.getSelected();
editor.getModel().getJs({ component });
```
Right now I'm using getJs from the Editor model, but in the next release you'll be able to do it directly from the editor instance `editor.getJs({ component })`
